lit_nlp/client/modules/shared_styles.css

```diff
@@ -1,7 +1,7 @@
 /**
  * Shared styles for all modules.
  */
-button {
+button:not(.plain) {
```

Comment:
This enables the "small" style selection button, similar to the data table component.  That component doesn't import shared styles, but this one does so that it gets other styles like "dropdown."  This change is trying to avoid refactoring this and just make the minimal change to opt-opt.